modules/ROOT/pages/release-notes/release-notes507.adoc

```diff
@@ -16,66 +16,66 @@ TinyMCE 5.0.7 release adds improvements to the TinyMCE editor to improve the ove
 
 To try out the *Premium Skins and Icon Packs* start with this link:{plugindirectory}skins-and-icon-packs[dedicated product page] or check out the https://www.tiny.cloud/pricing[pricing page] to see all the flexible pricing options.
 
-For more information and demos on the new *Premium Skins and Icon Packs* refer to the full link:{baseurl}/enterprise/premium-skins-and-icon-packs/[documentation].
+For more information and demos on the new *Premium Skins and Icon Packs* refer to the full xref:enterprise/premium-skins-and-icon-packs/index.adoc[documentation].
 
 === Tiny Drive
 
-*Tiny Drive* 1.3.0 comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/tinydrive/changelog/[changelog].
+*Tiny Drive* 1.3.0 comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:tinydrive/changelog.adoc[changelog].
 
 To try out *Tiny Drive* start with this https://www.tiny.cloud/drive/[dedicated product page] or check out the https://www.tiny.cloud/pricing[pricing page] to see all the flexible pricing options.
 
-For more information on *Tiny Drive* refer to the full link:{baseurl}/tinydrive/[documentation].
+For more information on *Tiny Drive* refer to the full xref:tinydrive/index.adoc[documentation].
 
-There is also a demo provided to explore the *Tiny Drive* capabilities link:{baseurl}/tinydrive/introduction/#demo[here].
+There is also a demo provided to explore the *Tiny Drive* capabilities xref:tinydrive/introduction.adoc#demo[here].
 
 === Enhanced Media Embed
 
 *Enhanced Media Embed* 2.2.0 adds a new `mediaembed_inline_styles` setting allowing styles to be embedded inline, instead of requiring additional content css to be added to your site.
 
-For more information on *Enhanced Media Embed* refer to the full link:{baseurl}/plugins/premium/mediaembed/#mediaembed_inline_styles[documentation].
+For more information on *Enhanced Media Embed* refer to the full xref:plugins/premium/mediaembed.adoc#mediaembed_inline_styles[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.7 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version507june52019[changelog].
+The new TinyMCE 5.0.7 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version507june52019[changelog].
```

Comment:
This xref doesn't resolve as the `version507june52019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version507june52019]]
== 5.0.7 - 2019-06-05
```

Reply:
done